generators/Exercises/Hamming.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using Generators.Input;
+
+namespace Generators.Exercises
+{
+    public class Hamming : Exercise
+    {
+        protected override void UpdateCanonicalData(CanonicalData canonicalData)
+        {
+            foreach (var canonicalDataCase in CanonicalData.Cases)
+            {
+                canonicalDataCase.ExceptionThrown = canonicalDataCase.Expected is long ? null : typeof(ArgumentException);
+            }
```

Comment:
For cases like this, I think I'd almost rather have an "IsNumber" or even "IsWholeNumber" if we feel the need to distinguish for future exercises. That way we're not trying to guess if we converted it to an int behind the scenes. At least this way we know that if we see a number, go `long`. 

It just gets us one step further away from guessing types and feels a little more fluent, in my opinion. Food for thought anyway.